admin/metabox/class-metabox.php

```diff
@@ -977,7 +978,18 @@ private function get_replace_vars() {
 			'sitedesc',
 			'sep',
 			'page',
+			'currentdate',
 			'currentyear',
+			'currentmonth',
+			'currentday',
+			'post_year',
+			'post_month',
+			'post_day',
+			'author_first_name',
+			'author_last_name',
+			'permalink',
+			'postcontent',
```

Comment:
Can we use an underscore in this variable name?

Reply:
yeap @johannadevos just did, thanks :)